syntax = "proto3";

service Showtime {
    rpc GetAllShowtimes(EmptySchedule) returns (stream ScheduleItem) {}
    rpc GetMoviesByDate(Date) returns (ScheduleItem) {}
}

message ScheduleItem {
    string date = 1;
    repeated string movies = 2;
}

message Date {
    string date = 1;
}

message EmptySchedule { }